modules/_unstable/README.md

```diff
@@ -0,0 +1,44 @@
+# Canvas Kit Unstable
+
+This module is a sandbox for work in progress components.
+
+Since we have a rather rigorous process for getting components in at a production level, it can be
+valuable to make them available earlier for teams who need them during development. Many of our
+components are contributions from internal product teams. While these components can suit the
+contributing product's use case, it takes time to iterate on the API/functionality as we evaluate
+them in a more global context. The Unstable module allows us to do that as needed.
+
```

Comment:
Should we add any documentation around contributing to components in `unstable`? 

Should we be instructing in [CONTRIBUTING.md](https://github.com/Workday/canvas-kit/blob/master/CONTRIBUTING.md) that any new components being contributed go into `unstable` first? Or will we instruct contributors to move their component there during the PR process if we think it's not ready for primetime?

Reply:
Hmm. I think we should encourage communication for new components earlier than a pull request. That's simple enough for internal developers, but we don't really have a public communication channel outside of github for external contributors...

We we encourage or allow new component development to external contributors?

Reply:
I think it's unlikely/uncommon but possible. IMO not all new components need to start in `unstable` so maybe we skip this. I think we only move it over if there's a lot of debate/uncertainty over the PR.

Reply:
On second thought, maybe it's best to call out in CONTRIBUTING.md that some new components may be moved there while they're fleshed out, but not all need to be started there.